luminaire/model/window_density.py

```diff
@@ -479,6 +479,7 @@ def _call_scoring(self, df=None, target_metric=None, anomaly_scores_gamma_alpha=
         :param luminaire.model.lad_structural.LADStructuralModel agg_data_model: Prediction model for aggregated data.
         :param str detection_method: Selects between "kldiv" or "sign_test" distance method.
         :param attributes: Model attributes.
+        :param agg_data: Aggregated Data per day.
```

Comment:
Is it always going to be per day?

Reply:
This is a higher level aggregated data which is per day at this moment

Reply:
Got it, so the the comments are correct.